src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java

```diff
@@ -958,13 +946,24 @@ private boolean checkAndPrepareMovements() {
 			mov.setWard((Ward) jComboBoxDestination.getSelectedItem());
 			mov.setDate(thisDate);
 			mov.setRefNo(jTextFieldReference.getText());
+			quantities.add(mov.getQuantity());
```

Comment:
why is `quantities` an `ArrayList<Integer>` when `getQuantity()` returns an `int`?  Make the array the same type so there is no boxing required.

Reply:
@dbmalkovsky could you please suggest the code modification directly?

Reply:
Nevermind; I was wrong.